tests/test_bash_utils.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I prefer use `/usr/bin/env bash` instead of it

Reply:
+1